clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -52,6 +52,12 @@ let autoRefresh: boolean = true;
  *         description: Signals fides.js to use the latest custom-fides.css (if available)
  *         schema:
  *           type: boolean
+ *       - in: query
+ *         name: gpp
+ *         required: false
+ *         description: Forces the GPP extension to be included in the bundle if the experience has GPP enabled
```

Comment:
Is this true? Doesn't this force GPP on even if the experience doesn't have GPP enabled...?

Reply:
good catch! fixed.